tests/draft-next/propertyDependencies.json

```diff
@@ -0,0 +1,112 @@
+[
+    {
+        "description": "propertyDependencies doesn't act on non-objects",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
+            }
+        },
+        "tests": [
+            {
+                "description": "ignores booleans",
+                "data": true,
+                "valid": true
+            },
+            {
+                "description": "ignores integers",
+                "data": 123,
+                "valid": true
+            },
+            {
+                "description": "ignores floats",
+                "data": 1.0,
+                "valid": true
+            },
+            {
+                "description": "ignores string",
+                "data": "abc",
+                "valid": true
+            },
+            {
+                "description": "ignores arrays",
+                "data": [],
+                "valid": true
+            },
+            {
+                "description": "ignores null",
+                "data": null,
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "propertyDependencies doesn't act on non-string property values",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
+            }
+        },
+        "tests": [
+            {
+                "description": "ignores booleans",
+                "data": {"foo": false},
+                "valid": true
+            },
+            {
+                "description": "ignores integers",
+                "data": {"foo": 2},
+                "valid": true
+            },
+            {
+                "description": "ignores floats",
+                "data": {"foo": 1.1},
+                "valid": true
+            },
+            {
+                "description": "ignores objects",
+                "data": {"foo": {}},
+                "valid": true
+            },
+            {
+                "description": "ignores objects wth a key of the expected value",
+                "data": {"foo": {"bar": "baz"}},
+                "valid": true
+            },
+            {
+                "description": "ignores objects with the expected value nested in structure",
+                "data": {"foo": {"baz": "bar"}},
+                "valid": true
+            },
+            {
+                "description": "ignores arrays",
+                "data": {"foo": []},
+                "valid": true
+            },
+            {
+                "description": "ignores null",
+                "data": {"foo": null},
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "disallowing values",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
```

Comment:
It seems like it'd be nice to have at least one test which uses an object schema rather than just a boolean, lest someone accidentally think this is something that simply takes a boolean flag.